### DISCUSSION TITLE
Should not the logic be opposite?

### DISCUSSION THREAD

**Discussion by metelik** [bacnet-stack/bacnet-stack#618] [Q&A]
I was wondering if the transitions' bits are set a recipient should be considered active or those work as a mask?
ref.:
https://github.com/bacnet-stack/bacnet-stack/blob/7b6dcada60457a4b891f38fa10a12f4302f0f7cb/src/bacnet/basic/object/nc.c#L498

In the SHRAE 135-2020 table 12-25:
"Transitions BACnet Event Transition Bits A set of three flags that indicate those transitions
{TO_OFFNORMAL, TO_FAULT, TO_NORMAL} for
which this recipient is suitable:

**Comment by skarg** [comment]
Changed via pull request [#623] - thanks!
